REFERENCE.md

```diff
@@ -1891,17 +1891,29 @@ Returns: `Any` encloses the ipv6 addresses with square brackets.
 
 ### <a name="ensure_packages"></a>`ensure_packages`
 
-Type: Ruby 3.x API
+Type: Ruby 4.x API
 
 It optionally takes a hash as a second parameter that will be passed as the
 third argument to the ensure_resource() function.
 
-#### `ensure_packages()`
+#### `ensure_packages(Variant[Hash,String[1],Array[String[1]]] $packages, Optional[Hash] $default_attributes)`
 
 It optionally takes a hash as a second parameter that will be passed as the
 third argument to the ensure_resource() function.
 
-Returns: `Any` install the passed packages
+Returns: `Any`
```

Comment:
Is there are return type for this?

Reply:
Is there a way to specify that the function doesn't return anything useful?